src/e2e/resources/data/InstructorSessionIndividualExtensionPageE2ETest.json

```diff
@@ -0,0 +1,219 @@
+{
+  "accounts": {
+    "instructor1": {
+      "googleId": "tm.e2e.ISesIe.instructor1",
+      "name": "Instructor 1",
+      "isInstructor": true,
+      "email": "tmms.instructor1@gmail.tmt",
+      "institute": "TEAMMATES Test Institute 1"
+    },
+    "instructor2": {
+      "googleId": "tm.e2e.ISesIe.instructor2",
+      "name": "Instructor 2",
+      "isInstructor": true,
+      "email": "tmms.instructor2@gmail.tmt",
+      "institute": "TEAMMATES Test Institute 1"
+    }
+  },
+  "courses": {
+    "course": {
+      "id": "tm.e2e.ISesIe.CS2104",
+      "name": "Programming Language Concepts",
+      "institute": "TEAMMATES Test Institute 1",
+      "timeZone": "Asia/Singapore"
+    }
+  },
+  "instructors": {
+    "ISesIe.instructor1": {
+      "googleId": "tm.e2e.ISesIe.instructor1",
+      "courseId": "tm.e2e.ISesIe.CS2104",
+      "name": "Instructor 1",
+      "email": "instructor1.tmms@gmail.tmt",
```

Comment:
This does not match up with the one in accounts. Same for the other instructor and same for student Danny in the feedback responses. Not sure if it's worth bothering with though since the test cases still pass.

Reply:
Thanks for spotting this, yes this was a mistake on my part.